src/lib/decl-processor.js

```diff
@@ -19,7 +19,7 @@ const prepareAsset = paths.prepareAsset;
  * @type {UrlRegExp[]}
  */
 const URL_PATTERNS = [
-    /(url\(\s*['"]?)([^"')]+)(["']?\s*\))/g,
+    /(url\(\s*['"]?)(?!~)([^"')]+)(["']?\s*\))/g,
```

Comment:
let's do it differently
add condition for `~` to here
https://github.com/postcss/postcss-url/blob/master/src/lib/paths.js#L37

Reply:
@sergcen done!